src/Services/TaskHandler.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+
+namespace App\Services;
+
+use App\Entity\Task;
+use App\Repository\UserRepository;
+use App\Repository\ProjectRepository;
+use Doctrine\ORM\EntityManagerInterface;
+
+
+class TaskHandler
+{
+    /**
+     * @var UserRepository
+     */
+    private $userRepository;
+
+    /**
+     * @var ProjectRepository
+     */
+    private $projectRepository;
+
+    private $em;
+
+    public function __construct(
+        ProjectRepository $projectRepository,
+        UserRepository $userRepository,
+        EntityManagerInterface $em
+    ) {
+
+        $this->userRepository = $userRepository;
+        $this->projectRepository = $projectRepository;
+        $this->em = $em;
+    }
+
+    public function validate($data, ?Task $task = null): bool
+    {
+        $repository = $this->em->getRepository(Task::class);
```

Comment:
either inject this repository too, either don't inject user and project repositionies